examples/imx500/rpk_update_network_intrinsics

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
```

Comment:
Maybe this file should live in the imx500-models repo?

Reply:
That works for me, I'll remove it from this commit.